index.html

```diff
@@ -5,9 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Jobiz</title>
-
     <link rel="stylesheet" href="/src/assets/css/global.css">
+    <script async type="module" src="src/js/localstorage.js"></script> <!-- async so the localstorageTest will have priority-->
+    <!-- <script defer type="module" src="src/js/main.js"></script>        defer so the main.js will load symultanously -->
```

Comment:
Why this code is commented? Do we need it? If we need it then why we commented this? File `main.js` do not exist. Nobody but you will know the answers to these questions and nobody will know if it is save to delete those comments.
In general do not leave any comments unless it is extreamly necessary and probably in all cases you won't need to use comments as I explained when you can use comments in previous comments.
Do we really need to tell what `defer` or `async` do? If anyone don't know it then he will check the documentation. Later this comment can even move to the wrong place and cause false information.
When developing a group project you need to think about other teammates. They will read it and waste time on those comments.

Reply:
Comment deleted.